src/App.svelte

```diff
@@ -4,15 +4,15 @@
 
   let grossIncome = 0;
   let year = '2022';
-  let marrStatus = 'single';
+  let marrStatus;
```

Comment:
This is a bit nitppicky, because it seems we'll default to the first status in the dropdowns anyways, but can you put the hardcoding of marrStatus = 'single'? I'd prefer the code to explicitly state that single is the default just in case anything changes down the line.